src/genie/libs/parser/ironware/tests/ShowMPLSVLL/cli/equal/normal_output_expected.py

```diff
@@ -0,0 +1,36 @@
+expected_output = {
+  'vll': {
+    'MY-FIRST-VLL': {
+      'vcid': 2456,
+      'vll_index': 2,
+      'local': {
+        'type': 'tagged',
+        'interface': 'ethernet2/5',
+        'vlan_id': 3043,
+        'state': 'Up',
+        'mct_state': 'None',
+        'ifl_id': '--',
+        'vc_type': 'tag',
+        'mtu': 9190,
+        'cos': '--',
+        'extended_counters': True,
+        'counters': False
+      },
+      'peer': {
```

Comment:
if you might have multiple peers, you might want to have like below:
```
'peer': {
    '192.168.1.1': {
        'state': 'UP',

Reply:
@tahigash A VLL is a point-to-point L2VPN pseudowire, it will never have more than one peer. Going to keep as is for now.

Reply:
ok. if so, no issue. thanks for the confirmation!